cloudsync_onedrive.py

```diff
@@ -460,13 +479,10 @@ def _fetch_sites(self):
 
     def _fetch_drives_for_site(self, site: Site):
         if not site.is_cached:
-            try:
-                drives = self._direct_api("get", f"/sites/{site.id}/drives").get("value", [])
-                drives.sort(key=lambda sd: sd["name"].lower())
-                for drive in drives:
-                    self._save_drive_info(site, drive)
-            except Exception as e:
-                log.warning("failed to get site drive info: %s", repr(e))
+            drives = self._safe_direct_api(f"/sites/{site.id}/drives", default={}).get("value", [])
```

Comment:
This previously swallowed all errors, now this can raise Disconnected and Token errors. Possibly an improvement... have the callers of this method been evaluated to confirm that these (possibly new) exception types are handled?

Reply:
should be all set here, this function is part of the "fetch namespace info" flow, and raises the same errors as the rest